### PR TITLE
Add styles to the sources pane

### PR DESCRIPTION
**Pull request by jasonLaster**
Fun PR for styling the sources pane

A lot of the CSS is a place holder for the tree view

screen shot 2016-05-02 at 7 14 00 pm

### CODE REVIEW

File: public/js/test/fixtures.json [hunk 1]
```diff
@@ -4,12 +4,14 @@
...
-        "filename": "foo.js"
+        "filename": "foo.js",
+        "pathname": "foo/foo.js"
       },
       "barSourceActor": {
         "actor": "barSourceActor",
         "url": "http://example.com/bar/bar.js",
-        "filename": "bar.js"
+        "filename": "bar.js",
+        "pathname": "bar/bar.js"
```
Comment (jasonLaster) [inline public/js/test/fixtures.json]: i feel guilty adding filename and pathname to the fixtures. should really be a query

Comment (jlongster) [inline public/js/test/fixtures.json]: Haha :)

**Comment by jlongster** [comment]
I'm a little unsure of the `<>` icon, what is that supposed to mean?

If these are coming from Helen's designs, I don't think those were very final. Just worth mentioning. Feel free to merge.

Also, I was thinking of not having the filtering UI at all and maybe just a "Sources" header because we weren't going to do filtering. And if we are, I'm not sure we should dedicate that whole area to it... We should probably just do the Cmd+P thing

**Comment by jasonLaster** [comment]
> <> icon 

that's coming from the designs. I think that can be added and then tweaked

> filtering UI

I'll take that out so we can think through the UX

**Comment by jasonLaster** [comment]
> <> icon

my guess is we'll likely want to show a file icon like "atom", "sublime", "chrome dt"

**Comment by jasonLaster** [comment]
merged

**Pull request closed without merging**